### ISSUE TITLE
add description to uploaded photos

### ISSUE THREAD

**Issue by anthonymartin** [popcorndao/workspace#202]
All additional uploaded photos should have alt text descriptions to go with them. This is important for universal accessibility.

The default alt text for profile and header images should be: `{organization} profile photo` and `{organization} header photo`

**Comment by RedVeil** [comment]
PR https://github.com/popcorndao/workspace/pull/219

**Comment by anthonymartin** [comment]
Can you resolve the conflicts? Will be good to merge after that
